docs/reference/commands/node-tool.asciidoc

```diff
@@ -206,6 +216,44 @@ The message `Node was successfully detached from the cluster` does not mean
 that there has been no data loss, it just means that tool was able to complete
 its job.
 
+
+[[node-tool-repurpose]]
+[float]
+==== Repurposing nodes
+
+A master-eligible and data carrying node holds on-disk index data:
```

Comment:
I think I'd prefer to try and split this whole section into two parallel descriptions about `node.data` and `node.master` respectively, because here it's a bit unclear what this says about a master-ineligible data node or (conversely) a dedicated master node. It might be a little repetitive, but I'm ok with that.